llvm/lib/Target/DirectX/DXIL.td

```diff
@@ -47,6 +47,7 @@ def ResRetInt32Ty : DXILOpParamType;
 def HandleTy : DXILOpParamType;
 def ResBindTy : DXILOpParamType;
 def ResPropsTy : DXILOpParamType;
+def ResSplitDoubleTy : DXILOpParamType;
```

Comment:
This should just be "SplitDoubleTy". The types that are prefixed with "Res" are that way to match the actual type names in DXIL.